libraries/AP_Param/AP_Param.cpp

```diff
@@ -2201,6 +2202,9 @@ bool AP_Param::read_param_defaults_file(const char *filename, bool last_pass)
             }
             continue;
         }
+        if (idx >= param_overrides_len) {
+            INTERNAL_ERROR(AP_InternalError::error_t::flow_of_control);
```

Comment:
Do we really want to just rumble on if we get this?

Reply:
fixed